webpack/components/extensions/Hosts/BulkActions/BulkRepositorySetsWizard/03_ReviewFooter.js

```diff
@@ -0,0 +1,73 @@
+import React, { useContext } from 'react';
+import { useDispatch } from 'react-redux';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  Button,
+  WizardFooterWrapper,
+  useWizardContext,
+} from '@patternfly/react-core';
+import { BulkRepositorySetsWizardContext } from './BulkRepositorySetsWizard';
+import { bulkUpdateHostContentOverrides } from './actions';
+import { pendingOverrideToApiParamItem } from './helpers';
+
+export const BulkRepositorySetsReviewFooter = () => {
+  const {
+    pendingOverrides,
+    finishButtonText,
+    allStepsValid,
+    finishButtonLoading,
+    setFinishButtonLoading,
+    closeModal,
+    hostsBulkSelect,
+  } = useContext(BulkRepositorySetsWizardContext);
+
+  const { goToStepById } = useWizardContext();
+  const dispatch = useDispatch();
+
+  const overridesEntries = Object.entries(pendingOverrides);
+  const apiParams = overridesEntries
+    .map(([repoLabel, value]) => pendingOverrideToApiParamItem({ repoLabel, value }))
+    .filter(item => item);
+
+  const saveContentOverrides = () => {
+    const requestBody = {
+      included: {
+        search: hostsBulkSelect.fetchBulkParams(),
+      },
+      content_overrides: apiParams,
+    };
+    dispatch(bulkUpdateHostContentOverrides(
+      requestBody, hostsBulkSelect.fetchBulkParams(),
```

Comment:
Doesn't request body already include these? Why do need  a separate `hostsBulkSelect.fetchBulkParams()` ..  After all `bulkUpdateHostContentOverrides` includes both params and bulk params and just concats them to the same list ?